### PR TITLE
Added sections on the landing page

### PR DESCRIPTION
**Pull request by NB134**
Added features, about and contribute sections on the landing page
Content was taken from readme.md.
Working demo link:  https://dash-three.vercel.app/.
Give feedback on the design and dontent of the added sections.

### CODE REVIEW

File: src/scss/login.module.scss [hunk 1]
```diff
@@ -3,10 +3,12 @@ $tiber: #09383e;
 $elephant: #0c383e;
 $white: #fff;
 
+
```
Comment (kartikcho) [inline src/scss/login.module.scss]: nit: extra committed line.

File: src/scss/login.module.scss [hunk 2]
```diff
@@ -121,10 +123,12 @@ $white: #fff;
   margin: 0 auto;
   position: relative;
   top: 1rem;
-  background: url(https://i.giphy.com/media/l0iVAuEtPF5KRbqmvd/source.gif);
-  background-size: contain;
-  background-position: center;
+  // background: url(https://i.giphy.com/media/l0iVAuEtPF5KRbqmvd/source.gif);
```
Comment (kartikcho) [inline src/scss/login.module.scss]: Could you remove this since we've agreed on having our logo on landing.

File: src/scss/login.module.scss [hunk 3]
```diff
@@ -252,7 +256,7 @@ $white: #fff;
 %button,
 .button {
   padding: 2px 8px 0 10px;
-  border-radius: 5px;
+
```
Comment (kartikcho) [inline src/scss/login.module.scss]: nit: A lot of extra lines seems to have been committed, could you run the file through a basic formatter?

File: src/scss/login.module.scss [hunk 4]
```diff
@@ -279,11 +284,379 @@ $white: #fff;
...
+.btnSlideEffect:hover,
+.btnSlideEffect:focus {  
+  animation: pulse 1s;
+  background-color: #3c4b64;
+  color:#ebedef;
+  box-shadow: 0 0 0 2em rgba(#3c4b64,0);
+}
+
+// /////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Landing Page Sections CSS
```
Comment (kartikcho) [inline src/scss/login.module.scss]: Could you remove these lines 303-304.

File: src/scss/login.module.scss [hunk 5]
```diff
@@ -279,11 +284,379 @@ $white: #fff;
...
+  height:100%;
+  position: relative;
+  padding:1em;
+  font-size:14px;
+  margin-top:8em;
+  }
+
+ 
+  // Feature Section
+  .Features
```
Comment (kartikcho) [inline src/scss/login.module.scss]: Let's ensure all classes use small case and hyphens for naming. Change this to `features` and same for other affected classes.

File: src/scss/login.module.scss [hunk 6]
```diff
@@ -279,11 +284,379 @@ $white: #fff;
...
+    height:100%;
+		
+  }
+
+  .Features:nth-child(1)
+    {
+        margin-top:0;
+    } 
+
+  .TextContainer{
```
Comment (kartikcho) [inline src/scss/login.module.scss]: same here as above, use hyphens to make it `text-container`

File: src/scss/login.module.scss [hunk 7]
```diff
@@ -279,11 +284,379 @@ $white: #fff;
...
+  // AboutUs Section
+
+  .AboutUs > .GifCard > img{
+    width:70%;
+    box-shadow:none;
+  }
+
+
+  // Contribute Section
+.Contri{
```
Comment (kartikcho) [inline src/scss/login.module.scss]: Could you rename this to `contribute`, would be more easy to understand.

File: src/scss/login.module.scss [hunk 8]
```diff
@@ -279,11 +284,379 @@ $white: #fff;
...
+  width:50%;
+}
+
+.BtnContainer{
+  display:flex;
+  flex-direction: row;
+  gap:0;
+}
+
+.ContriButton{
```
Comment (kartikcho) [inline src/scss/login.module.scss]: Similarly here, `contribute-button` or `contribute-btn`

File: src/scss/login.module.scss [hunk 9]
```diff
@@ -279,11 +284,379 @@ $white: #fff;
...
+    position: relative;
+  }
+
+  .IntroText li, .IntroText p{
+    line-height: 1.5em;
+  }
+
+  // AboutUs Section
+
+  .AboutUs > .GifCard > img{
```
Comment (kartikcho) [inline src/scss/login.module.scss]: There is no About Us section on the page so let's make this `about` or something similar. Same for the comment on 383.

File: src/views/pages/login/Login.js [hunk 10]
```diff
@@ -54,6 +56,138 @@ const Login = ({ authenticate }) => {
           </div>
         </div>
       </div>
+    </div>
+
+{/* Features Section */}
```
Comment (kartikcho) [inline src/views/pages/login/Login.js]: Let's remove these comments and run this file through a formatter too.

**Review by kartikcho — CHANGES_REQUESTED** [review]
Hi, thanks for working on this!

I see some assets that have been added but not used, could you remove them from the PR?

Run both the files through a formatter like Pretttier to have consistent formatting, also remove comments from `login.js`.

Good work!

**Comment by kartikcho** [comment]
Requesting a review from @sauravhiremath to review page content and check the code in general.

**Comment by NB134** [comment]
I ran prettier and removed unused assets from old commits.

**Review by sauravhiremath — COMMENTED** [review]
@NB134 That's some solid work!

There are a few changes though
* Remove the styled CSS import and just use:
```javascript
import '/theme/login.scss'
```
* Add escape characters for the latitudes and longitudes

I am mentioning them as bullet points here, as all the changes right now are being migrated to [bundly/dash-beta](https://github.com/bundly/dash-beta).

If possible could you open the PR with the same changes on the `dash-beta` repo? It should be relatively easier to import these changes as the files structure are very similar, and we can help you out if needed during migration.

Anyway, this is awesome work!

**Comment by NB134** [comment]
So, i need to remove the styled css import and import the css normally, changing the className value from {styled.class} to "class".
Also i have added latitudes and longitudes using content property of the pseudo element so should i change it entirely into unicode or only spaces in unicode.

**Comment by sauravhiremath** [comment]
1. Yeah, just importing CSS as you mentioned
2. Just the apostrophes and degrees in Unicode, like `&quot;`, `&apos;` and `&deg;`. No need to encode `spaces`

**Pull request closed without merging**